src/DynamoCore/Graph/Notes/NoteModel.cs

```diff
@@ -11,6 +11,14 @@ namespace Dynamo.Graph.Notes
     /// </summary>
     public class NoteModel : ModelBase
     {
+        /// <summary>
+        /// 
+        /// </summary>
+        public event Action<ModelBase> UndoRequest;
```

Comment:
in general, make the access modifiers for members as conservative as possible (internal, private etc) - if you make things public we need to maintain them until the next major release - this is very difficult.

Reply:
I was following how the properties were defined in the class. I see your point, changed to internal.